src/js/components/SharedComponents/Modal.jsx

```diff
@@ -0,0 +1,31 @@
+/**
+  * Modal.jsx
```

Comment:
Do you think we will use this modal somewhere else? If it is a one time thing it should stay in the related directory. if it is something that will be reused then this is fine. Nothing is wrong per se, just something to keep in mind when creating new files.